### DISCUSSION TITLE
Static Dashboard instead of dynamic dashboard

### DISCUSSION THREAD

**Discussion by kbommaSFDC** [jongpie/NebulaLogger#248] [General]
Hello Experts,
We have been using Nebula framework for one of our client and trying to upgrade the framework to version 36.
In this new release new dynamic dashboard has been included but our client org reached to max dynamic dashboards (10) so we are not able to upgrade without increasing the dynamic dashboard limit. 

We would like to check with the community, whether dashboard can be converted to static dashboard (Run as specified user) instead of dynamic dashboard (Run as logged in user). 

Currently max dynamic dashboards for the Org is 10 only. Thank you for considering this request.

**Comment by jongpie** [comment]
Hi again @kbomma-sf - just wanted to make sure I answered your question here too, I'm planning to resolve this issue later this week/this weekend, you can track the status via issue #247. I had forgotten about the org limit of 10 dynamic dashboards, so I'm going to remove the new dashboard altogether, and instead make it an optional add-on (available as a separate unlocked package).

**Comment by kbommaSFDC** [comment]
Thank you Jonathan for considering our request. Appreciate your guidance and contribution to the community.
